website/docs/components/table/partials/code/how-to-use.md

```diff
@@ -545,6 +545,67 @@ In this case the table layout is still set to `auto` (default). If instead you w
 </div>
 ```
 
+### Multi-select table
+
+A multi-select table includes checkboxes enabling users to select multiple rows in a table for purposes of performing bulk operations. Checking or unchecking the checkbox in the table header either selects or deselects the checkboxes on each row in the table body. Individual checkboxes in the rows can also be selected or deselected.
```

Comment:
@andgen404, @KristinLBradley has done a great job summarizing what this feature offers. You may consider replacing your summary on the Guidelines tab with this.